### PR TITLE
mark translatesfx as deprecated

### PR DESCRIPTION
**Pull request by atoulme**
**Description:**
Add a notice statement when executing translatesfx and a few deprecation notices on the translatesfx tooling to indicate that it is now deprecated and will eventually be removed.

### CODE REVIEW

**Comment by theletterf** [comment]
@atoulme We're currently recommending translatesfx in https://docs.splunk.com/observability/en/gdi/opentelemetry/translation-tool.html

Happy to add that deprecation note to the docs, too, if needed. Just say the word.

File: cmd/translatesfx/README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # SignalFx Smart Agent Configuration Translation Tool
 
+> # :warning: End of Support (EoS) Notice
+> **The SignalFx Smart Agent has reached End of Support.**
+>
+> The [Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/opentelemetry.html) is the successor. Smart Agent monitors are available and supported through the [Smart Agent receiver](https://docs.splunk.com/Observability/gdi/opentelemetry/components/smartagent-receiver.html) in the Splunk Distribution of OpenTelemetry Collector.
+>
+> To learn how to migrate, see [Migrate from SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html).
```
Comment (rmfitzpatrick) [inline cmd/translatesfx/README.md:8]: this tool is part of the migration process so this guidance is difficult to reconcile as is imo.

```suggestion
```

**Comment by atoulme** [comment]
@theletterf appreciate, please do if possible.

File: cmd/translatesfx/README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # SignalFx Smart Agent Configuration Translation Tool
 
+> # :warning: End of Support (EoS) Notice
+> **The SignalFx Smart Agent has reached End of Support.**
+>
+> The [Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/opentelemetry.html) is the successor. Smart Agent monitors are available and supported through the [Smart Agent receiver](https://docs.splunk.com/Observability/gdi/opentelemetry/components/smartagent-receiver.html) in the Splunk Distribution of OpenTelemetry Collector.
+>
+
```
Comment (rmfitzpatrick) [inline cmd/translatesfx/README.md:8]: May be helpful to not require running the tool or seeing the updated product doc when it lands for good measure:

```suggestion

## Notice: `translatesfx` is deprecated and will be removed from the distribution of the collector in an upcoming release.
```

Comment (theletterf) [reply cmd/translatesfx/README.md:8]: Oh, so we'd be removing the tool entirely in 0.90?

Comment (atoulme) [reply cmd/translatesfx/README.md:8]: No, in an "upcoming release".

**Review by rmfitzpatrick — APPROVED** [review]

**Pull request merged**